src/components/SQFormGuidedWorkflow/PropTypes.js

```diff
@@ -108,5 +108,7 @@ export const GuidedWorkflowProps = {
   onError: PropTypes.func.isRequired,
   /** Task Module configuration Object(s) */
   taskModules: PropTypes.arrayOf(PropTypes.shape(TaskModuleProps).isRequired)
-    .isRequired
+    .isRequired,
+  /** An object of css-in-js style properties to be passed*/
```

Comment:
Our Storybook depends on a properly formatted JS Doc comment.
A space is missing between passed and */